Caddyfile

```diff
@@ -1,16 +1,11 @@
+{
+	acme_ca https://acme-v02.api.letsencrypt.org/directory
+}
+
 testnet-node-1.cere.network:9945 {
-	log stdout
-	errors stderr
-	gzip
-	tls ab@cere.io
-	proxy / boot_node:9944 {
-	     websocket
-	  }
+	reverse_proxy / boot_node:9944
```

Comment:
@groschovskiy 
Where can we find caddy's usual and error logs?

Reply:
Via docker-compose for current moment